plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -122,15 +123,15 @@ tests =
       ]
   , goldenWithEval ":kind! treats a multilined result properly" "T24" "hs"
   , goldenWithEval ":kind treats a multilined result properly" "T25" "hs"
-  , goldenWithEval "local imports" "T26" "hs"
+  , goldenWithEvalAndFs "local imports" (FS.directProjectMulti ["T26.hs", "Util.hs"]) "T26" "hs"
   , goldenWithEval "Preserves one empty comment line after prompt" "T27" "hs"
```

Comment:
Why did you leave some of these as they are? lack of time?

Reply:
`goldenWithEval` is a local function. It is redefined to create a direct cradle project, so all of the tests are using the VirtualFileTree.